src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -41,3 +65,62 @@ export class SurgingTempest extends Analyser {
 		return (statusUptime / fightUptime) * 100
 	}
 }
+
+export class StormsEye extends CoreGauge {
+	static override handle = 'stormseye'
+
+	@dependency private suggestions!: Suggestions
+
+	private surgingTempest = this.add(new TimerGauge({
+		maximum: 60,
+	}))
+
+	private earlyRefreshCount = 0
+
+	override initialise() {
+		super.initialise()
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_GENERATORS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_GENERATION_AMOUNT)
```

Comment:
Prefer not to put the placeholder assignment here, we just use `()` outright if there's no args being used. I'm guessing this is a holdover from either C# or Go.

Reply:
C# <_< >_>